docs/user-guide/data-structures/data-structures.ipynb

```diff
@@ -106,72 +108,24 @@
     "var.values"
    ]
   },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "print(var.variances)"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "Variances must have the same shape as values, and units are specified using the [scipp.units](../reference/units.rst) module or with a string:"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "var = sc.array(\n",
-    "    dims=['x', 'y'],\n",
-    "    unit='m/s',\n",
-    "    values=np.random.rand(2, 4),\n",
-    "    variances=np.random.rand(2, 4),\n",
-    ")\n",
-    "sc.show(var)"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "var"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "var.variances"
-   ]
-  },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "### 0-D variables (scalars)\n",
     "\n",
-    "A 0-dimensional variable contains a single value (and an optional variance).\n",
-    "The most convenient way to create a scalar variable is by multiplying a value by a unit:"
```

Comment:
Why removed?

Reply:
I wanted to keep this section brief because creation is explained on the new page. I don't mind duplicating some information. But I don't think we should duplicate all of it.